src/app/portlets/dot-edit-content/dot-edit-content.component.spec.ts

```diff
@@ -60,4 +74,20 @@ describe('DotEditContentComponent', () => {
     it('should be created', () => {
         expect(component).toBeTruthy();
     });
+
+    it('should show dotLoadingIndicatorService on init', () => {
+        const spyLoadingIndicator = spyOn(component.dotLoadingIndicatorService, 'show');
+
+        component.ngOnInit();
+
+        expect(spyLoadingIndicator).toHaveBeenCalled();
+    });
+
+    it('should hide dotLoadingIndicatorService when the component loads', () => {
+        const spyLoadingIndicator = spyOn(component.dotLoadingIndicatorService, 'hide');
+
+        component.onLoad(Event);
+
+        expect(spyLoadingIndicator).toHaveBeenCalled();
+    });
```

Comment:
We need a one more test to check if the dot-loading-indicator tag is in the template